### PR TITLE
PCG-16968 | Dotnet upgrade from version 6 to 8

### PR DESCRIPTION
**Pull request by rajlaxmiSophos**

### CODE REVIEW

File: FlowLogsStage1/Stage1BlobTrigger.cs [hunk 1]
```diff
@@ -39,8 +41,18 @@ public static void Run(
...
+                string storageConnectionString = Util.GetEnvironmentVariable("AzureWebJobsStorage");
+                // Create a TableClient instance
+                TableClient tableClient = new TableClient(storageConnectionString, "checkpoints");
+                // Create table if not exist
+                await tableClient.CreateIfNotExistsAsync();
+
                 // get checkpoint
-                Checkpoint checkpoint = Checkpoint.GetCheckpoint(blobDetails, checkpointTable);
+                Checkpoint checkpoint = await Checkpoint.GetCheckpoint(blobDetails, tableClient);
+                // break up the block list into 10k chunks
```
Comment (sachinkumar07) [inline FlowLogsStage1/Stage1BlobTrigger.cs]: remove duplicate comment

File: README.md [hunk 2]
```diff
@@ -1,5 +0,0 @@
-# Sophos-Cld-Azure-FlowLogs-Powershell-Functions
-
```
Comment (sachinkumar07) [inline README.md:2]: Why are we removing this file?

Comment (rajlaxmiSophos) [reply README.md:2]: Reverted this change

File: FlowLogsStage1/Stage1BlobTrigger.cs (hunk 1 above)
Comment (rajlaxmiSophos) [reply FlowLogsStage1/Stage1BlobTrigger.cs]: Removed

**Review by sachinkumar07 — APPROVED** [review]

**Review by sparshsophos — APPROVED** [review]

**Review by aamir-stpl — APPROVED** [review]

**Review by prasoon124 — APPROVED** [review]

**Review by siddharthkundal — APPROVED** [review]

**Review by arun542 — APPROVED** [review]

**Review by apoorv-sophos — APPROVED** [review]

**Pull request merged**